### DISCUSSION TITLE
kbEsc, kbF1, kbF2 missing

### DISCUSSION THREAD

**Discussion by paule32** [magiblot/tvision#117] [Q&A] [answered]
Hello,
when I handleEvent I came to the Point, that the Compiler does not known/find the constants for kbEsc, and so on...
Is there a #define Uses ? or other include header ?
At this Time, I workaround with the values that i catch from the TVDemo EventViewer.

**Comment by magiblot** [comment, marked as answer]
Yes, you need `#define Uses_TKeys`
